### DISCUSSION TITLE
Support "jbang @kiegroup --help" to get a list of all scripts for a catalog repo

### DISCUSSION THREAD

**Discussion by ge0ffrey** [jbangdev/jbang#1101] [Q&A] [answered]
I can ask the help of a specific jbang script of the kiegroup repo:

`jbang dmn@kiegroup --help`

But how can I get a list of all the scripts for a catalog repo? It would be cool if this just works:

`jbang @kiegroup --help`

or `jbang *@kiegroup --help`

NOTE: With `jbang --help` I don't seem to see a command to gives me a list of all scripts in a catalog.

**Comment by maxandersen** [comment, marked as answer]
`jbang catalog list kiegrup`

`jbang alias list kiegroup`

should work and is in jbang --help:

```
Configuration:
  trust       Manage which domains you trust to run scripts from.
  alias       Manage aliases for scripts.
  template    Manage templates for scripts.
  catalog     Manage Catalogs of aliases.
  app         Manage scripts installed on the user's PATH as commands.
```

suggestion if we could have described them better?

**Reply by ge0ffrey** [reply]
It would be nice to have it as one of the example expressions at the top of the help file. Not sure if I am the exception or the average user when I was looking for that...

**Comment by quintesse** [comment]
Perhaps we should change "Manage" to "List, add or remove"?
